### PR TITLE
finish user story post my diary entry via slack to channel

### PR DESCRIPTION
**Pull request by AnsMann**

### CODE REVIEW

File: src/app/SlackResultList.js [hunk 1]
```diff
@@ -17,21 +17,22 @@ const SlackLogo = styled.img`
...
+  channels,
   searchInput,
   handleContactClick,
 }) {
-  const filteredContacts = filterContacts(userContacts, searchInput)
-  if (filteredContacts.length !== 0) {
-    return filteredContacts.map(contact => (
-      <SlackUser
+  const filteredResult = filterData(userContacts, channels, searchInput)
+  if (filteredResult.length !== 0) {
```
Comment (melmare) [inline src/app/SlackResultList.js:27]: Isn't that the same as just writing `filteredResult`?

File: src/app/services.js [hunk 2]
```diff
@@ -23,6 +23,22 @@ export function getContacts() {
...
+    {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+      },
+    }
+  )
+    .then(res => res.json())
+    .then(data => data.channels)
+}
```
Comment (melmare) [inline src/app/services.js:40]: You might consider writing a catch function to log something out if an error occurs.

**Review by melmare — COMMENTED** [review]
Keep on your great work :)

File: src/app/SlackResultList.js (hunk 1 above)
Comment (AnsMann) [reply src/app/SlackResultList.js:27]: unfortunately it doesn't work

File: src/app/services.js (hunk 2 above)
Comment (AnsMann) [reply src/app/services.js:40]: yes i add this point to my backlog

**Pull request merged**